tools/iptb-plugins/filecoin/local/localfilecoin.go

```diff
@@ -106,16 +106,52 @@ func init() {
 			}
 		}
 
+		if err := os.Mkdir(filepath.Join(dir, "bin"), 0755); err != nil {
+			return nil, fmt.Errorf("could not make dir: %s", err)
+		}
+
+		dst := filepath.Join(dir, "bin", filepath.Base(binPath))
+		if err := copyFileContents(binPath, dst); err != nil {
+			return nil, err
+		}
+
+		if err := os.Chmod(dst, 0755); err != nil {
+			return nil, err
+		}
+
```

Comment:
Just so reviewers understand why this is here:

This makes it so we grab a copy of the binary that is used. This will either be whatever is found in the path (not shown in this change), or was is provided by the `AttrFilecoinBinary` attr (option, also not shown). This makes sure that when we drop into a shell we are using the same binary that the node was initiated with.